en/app.ftl

```diff
@@ -418,35 +563,68 @@ profile-label-details-hide = Hide Details
 profile-label-click-to-copy = Click to copy
+# Deprecated
 profile-label-copy-confirmation = Alias copied to clipboard
+# Deprecated
+profile-label-copy-confirmation-2 = Mask copied to clipboard
 profile-label-copied = Copied!
 profile-label-blocked = Blocked
 profile-label-forwarded = Forwarded
 profile-label-cancel = Cancel
+# Deprecated
 profile-blocked-copy = { -brand-name-firefox-relay } will delete messages before they can reach your inbox when you select blocking for this alias.
+profile-blocked-copy-2 = { -brand-name-firefox-relay } will delete messages before they can reach your inbox when you select blocking for this mask.
+# Deprecated
 profile-forwarded-copy = { -brand-name-firefox-relay } will send messages to your inbox when you select forwarding for this alias.
+profile-forwarded-copy-2 = { -brand-name-firefox-relay } will send messages to your inbox when you select forwarding for this mask.
 profile-forwarded-note = Note:
 profile-forwarded-note-copy = Email (including attachments) larger than { email-size-limit } are not currently supported and will not be forwarded.
 profile-forwarded-note-copy-v2 = Email (including attachments) larger than { email-size-limit } will not be forwarded.
 profile-stat-label-blocked = Emails Blocked
 profile-stat-label-forwarded = Emails Forwarded
+# Deprecated
 profile-stat-label-aliases-used = Email aliases used
+profile-stat-label-aliases-used-2 = Email masks used
+# Deprecated
 profile-filter-search-placeholder = Search aliases
-# Filter on Relay aliases that are still active; i.e. that still forward received emails to your email address.
+profile-filter-search-placeholder-2 = Search masks
+profile-filter-category-button-label = Filter visible masks
+profile-filter-category-button-tooltip = Filter masks by subdomain and/or whether they are currently blocking incoming email
+profile-filter-category-title = Filter visible masks
+profile-filter-no-results = No masks match your selected criteria. <clear-button>Clear all filters.</clear-button>
+# Deprecated
 profile-filter-category-option-active-aliases-v2 = Forwarding aliases
-# Filter on Relay aliases that are inactive; i.e. that do not forward received emails to your email address, but block them instead.
+# Filter on Relay masks that are still active; i.e. that still forward received emails to your email address.
+profile-filter-category-option-active-masks = Forwarding masks
+# Deprecated
 profile-filter-category-option-disabled-aliases-v2 = Blocking aliases
-# Filter on Relay aliases that have a random name (<gibberish>@mozmail.com).
+# Filter on Relay masks that are inactive; i.e. that do not forward received emails to your email address, but block them instead.
+profile-filter-category-option-disabled-masks = Blocking masks
+# Filter on Relay aliases that block promotional emails. "Promo" is an English slang/shortened version of "Promotion".
+profile-filter-category-option-promo-blocking-aliases = Promo-blocking aliases
```

Comment:
Ah, I should've removed that first one, since it was a new string and hadn't been localised yet. It's part of the new strings, so no longer in this PR, but I removed it in https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/69.